src/tutorial/src/step-1/description.md

```diff
@@ -21,6 +21,29 @@ If you are an experienced developer coming from Vue 2 or other frameworks, there
 
 - You can also switch between SFC-mode or HTML-mode. The former will show code examples in <a target="_blank" href="/guide/introduction.html#single-file-components">Single-File Component</a> (SFC) format, which is what most developers use when they use Vue with a build step. HTML-mode shows usage without a build step.
 
+:::tip
```

Comment:
hide this on sfc:

```suggestion
:::tip {:class="'html'"}
```

Reply:
Your suggestion didn't work for me for some reason, but I wrapped the part inside <div class="html"> which seems to do the job.